src/components/SelectionList/SelectionList.tsx

```diff
@@ -1,18 +1,19 @@
 import { Text } from 'rebass'
-import styled from 'styled-components'
+import styled from 'styled-components/macro'
```

Comment:
Should we use `'styled-components/macro'`import everywhere? Or should we use `'styled-components'` here?

Reply:
I used to debug a UI glitch. Switching to macro is on the [task list for July](https://www.notion.so/adamazad/Swapr-UX-DX-Improvement-Plans-for-July-2022-v1-0-904ba420c12e4014831ea9e1ed05ae05). For now, I don't think we should make this PR any bigger in terms of the file list.